packages/gasket-plugin-nextjs/lib/index.d.ts

```diff
@@ -1,6 +1,6 @@
 import type { IncomingMessage, ServerResponse } from 'http';
-import type { NextConfig } from 'next/dist/next-server/server/config-shared';
-import type NextServer from 'next/dist/next-server/server/next-server';
+import type { NextConfig } from 'next';
```

Comment:
without the updated import intellisense doesn't work in gasket.config for nextConfig